builder/vmware/common/step_run.go

```diff
@@ -42,18 +42,19 @@ func (s *StepRun) Run(ctx context.Context, state multistep.StateBag) multistep.S
 			vncPassword := vncPasswordRaw.(string)
 
 			ui.Message(fmt.Sprintf(
-				"The VM will be run headless, without a GUI. If you want to\n"+
-					"view the screen of the VM, connect via VNC with the password \"%s\" to\n"+
-					"vnc://%s:%d", vncPassword, vncIp, vncPort))
+				"The virtual machine will be run headless, without a GUI.\n"+
+					"To view the virtual machine console, connect using VNC:\n\n"+
+					"Endpoint: \"vnc://%s:%d\"\n"+
+					"Password: \"%s\"", vncIp, vncPort, vncPassword))
 		} else {
-			ui.Message("The VM will be run headless, without a GUI, as configured.\n" +
-				"If the run isn't succeeding as you expect, please enable the GUI\n" +
-				"to inspect the progress of the build.")
+			ui.Message("The virtual machine will be run headless, without .\n" +
+				"a GUI. If the build is not succeeding as expected, enable\n" +
```

Comment:
I would maybe hint at the possibility to setup VNC as an alternative here?

Reply:
Addressed in https://github.com/hashicorp/packer-plugin-vmware/commit/d6a20f766cb77d80838cf62ada62edaa17649664.

Reply:
Actaully, reverted becasue the VNC details would not be available.